docs/www/mirromaker2-migration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Using MirrorMaker 2 with Redpanda
+order: 2
+---
+
+# Migrating data to Redpanda
+
+One of the more elegant aspects of Redpanda is that it is compatible with the [Kafka API and ecosystem](/docs/www/faq.md).
+So, when you want to migrate data from Kafka or replicate data between Redpanda clusters,
+MirrorMaker 2 tool, bundled in the [Kafka download package](https://kafka.apache.org/downloads), is a natural solution.
+
+In this article, we'll go through the process of configuring MirrorMaker to replicate from one Redpanda cluster to another.
+For all of the details on MirrorMaker and its options, check out the [Kafka documentation](https://kafka.apache.org/documentation/#georeplication).
+
+## Prerequisites
+
+To set up the replication, we'll need:
+
+- 2 Redpanda Clusters - You can set up these clusters in any deployment you like, including [Kubernetes](/docs/www/quick-start-kubernetes.md), [Docker](/docs/www/quick-start-docker.md), [Linux](/docs/www/quick-start-linux.md), or [MacOS](/docs/www/quick-start-macos.md).
+    The key points here are:
+    - Make sure that the IP addresses and ports on each cluster are accessible by MirrorMaker.
+
+- An instance of MirrorMaker - You can set up a separate system with MirrorMaker or run MirrorMaker on one of the Redpanda clusters.
+
+## Installing MirrorMaker
+
+MirrorMaker 2 is run by a shell script that is part of the Kafka download package.
+To install MirrorMaker 2 on the machine that you want to run the replication between the clusters:
+
+1. Download the latest [Kafka download package](https://kafka.apache.org/downloads).
+
+    For example:
+
+    ```
+    curl -O https://www.apache.org/dyn/closer.cgi?path=/kafka/2.8.0/kafka_2.13-2.8.0.tgz
+    ```
+
+2. Extract the files from the archive:
+
+    ```
+    tar -xvf kafka_2.13-2.8.0.tgz
+    ```
+
+## Create the MirrorMaker 2 config files
+
+MirrorMaker uses configuration files to get the connection details for the clusters.
+In this example, we'll assume the file structure:
+
+```
+.
+└── kafka_2.13-2.8.0
+    └── bin
+        └── connect-mirror-maker.sh
```

Comment:
@ivotron did we end up shipping mm2 ourselves ?

Reply:
no, iirc there was an open question regarding dependencies. mm2 basically needs all of kafka, so this would need to ship the kafka binaries. And depending on whether this should be 100% self-contained, then also the jvm

Reply:
Ah got it. Ok.